kinetic/distribution.yaml

```diff
@@ -5044,14 +5044,24 @@ repositories:
       url: https://github.com/ros-gbp/libg2o-release.git
       version: 2016.4.24-0
     status: maintained
-  libmynteye-release:
+  libmynteye:
+    doc:
+      type: git
+      url: https://github.com/harjeb/libmynteye.git
+      version: master
     release:
       packages:
       - libmynteye
       tags:
         release: release/kinetic/{package}/{version}
       url: https://github.com/harjeb/libmynteye-release.git
-      version: 0.0.1-0
+      version: 0.0.1-1
```

Comment:
There's no `0.0.1-1` in the release repository (https://github.com/harjeb/libmynteye-release), you can't just change this you need to rerun bloom.

I'd suggest closing this pull request, rerunning bloom (with the correct repository name, use `-e` to edit the settings before running if necessary) and letting bloom open a new pull request for you.